strings/sillytips.txt

```diff
@@ -45,3 +45,12 @@ Help.
 Maints.
 BZ stops or slows down Lings chem regeneration drastically, make sure to BZ flood the station when lings are confirmed!
 Admins always regret meme options in their polls.
+ERP stands for Emergent Role Play.
+Remember if you see a bug report it on Github.
+Have good ideas for the game? Well make a suggestion on Github, we have templates just for it!
+Role Play is fun. Try it sometimes.
+This is a MRP Server, no really.
+Clean bot has more to say after being emaged. Try it sometimes with your limited use Emags... It will be worth it!
+Bar sign can be emaged.
+Mood.
+Ok Boomer, is not a helpful thing to say when being bowinked about your wrong doing.
```

Comment:
Bwoinked, though I'm not sure about this one either.

Reply:
Bowinked, as its the sound of a bow like a cross/bow/, and a ink like pen and /ink/?

Reply:
what